Workbooks/Azure Active Directory Conditional Access/By continuous access evaluation (Preview)/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Sign-ins by Continuous access evaluation status",
+    "author": "Microsoft",
+    "galleries": [{ "type": "workbook", "resourceType": "microsoft.aadiam/tenant", "order": 100 }]
+}
```

Comment:
Please set the preview to true and use a deeplink to expose it to the customers initially. Get feedback and then make it public for all.

Reply:
Ack. Thanks for pointing it out.